server/task_executor.go

```diff
@@ -59,15 +59,9 @@ func runMigration(ctx context.Context, l *zap.Logger, server *Server, task *api.
 		mi.Version = schemaVersion
 		mi.Description = task.Name
 	} else {
-		repoFind := &api.RepositoryFind{
-			ProjectID: &task.Database.ProjectID,
-		}
-		repoInner, err := server.store.GetRepository(ctx, repoFind)
+		repoInner, err := findRepositoryByTask(ctx, server, task)
```

Comment:
we can just name it repo and remove repoOutter?

Reply:
Done.